src/cpp/session/modules/SessionFind.cpp

```diff
@@ -63,18 +63,6 @@ class LocalProgress : public boost::noncopyable
       nextUpdate_ = updateIncrement_;
    }
 
-   void addUnit()
```

Comment:
Oh, I didn't mean to delete this function, I think it's fine to have a convenience `addUnit()` function, I just thought we could avoid duplication by making the body `{ addUnits(1) }`. Either way, I don't have a strong opinion about keeping it or not.

Reply:
Ah, that makes sense. I'm happy with the way it is now (removed) so going to leave as is.